src/mongocxx/options/find.hpp

```diff
@@ -270,6 +294,7 @@ class MONGOCXX_API find {
     stdx::optional<std::string> _comment;
     stdx::optional<enum cursor_type> _cursor_type;
     stdx::optional<std::int32_t> _limit;
+    stdx::optional<std::int64_t> _max_await_time_ms;
```

Comment:
This depends on [CDRIVER-966](https://jira.mongodb.org/browse/CDRIVER-966).

It will be a straight forward on line change, just need to set the option on the `mongoc_cursor_t` before wrapping it.